frontend/src/openapi/models/featureSearchResponseSchema.ts

```diff
@@ -28,7 +28,7 @@ export interface FeatureSearchResponseSchema {
      */
     createdAt: string | null;
     /** User who created the feature flag */
-    createdBy?: FeatureSearchResponseSchemaCreatedBy;
+    createdBy: FeatureSearchResponseSchemaCreatedBy;
```

Comment:
will this always have a created by even in old flags? We did do some sort of patching of pre-existing flags, didn't we?